interface-definitions/load-balancing-wan.xml.in

```diff
@@ -0,0 +1,395 @@
+                  </constraint>
+                </properties>
+              </leafNode>
+              <tagNode name="test">
+                <properties>
+                  <help>Rule number</help>
+                  <valueHelp>
+                    <format>u32:0-4294967295</format>
+                    <description>Rule number</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 0-4294967295"/>
+                  </constraint>
+                </properties>
+                <children>
+                  <leafNode name="resp-time">
+                    <properties>
+                      <help>Ping response time (seconds)</help>
+                      <valueHelp>
+                        <format>u32:1-30</format>
+                        <description>Response time (seconds)</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="numeric" argument="--range 1-30"/>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="target">
+                    <properties>
+                      <help>Health target address</help>
+                      <valueHelp>
+                        <format>ipv4</format>
+                        <description>Health target address</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="ipv4-address"/>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="test-script">
+                    <properties>
+                      <help>Path to user defined script</help>
+                      <valueHelp>
+                        <format>txt</format>
+                        <description>Script in /config/scripts</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="script"/>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="ttl-limit">
+                    <properties>
+                      <help>Ttl limit (hop count)</help>
+                      <valueHelp>
+                        <format>u32:1-254</format>
+                        <description>Number of hops</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="numeric" argument="--range 1-254"/>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="type">
+                    <properties>
+                      <help>WLB test type</help>
+                      <completionHelp>
+                        <list>ping ttl user-defined</list>
```

Comment:
TTL

Reply:
We should not use upper and lowercase CLI options. I agree with @sever-sever to use `ttl`

Reply:
Ah, sorry, here I clicked in a wrong place. Since it's a completion help, it should indeed use the lower case like the actual option. Sorry for the confusion!